Lib/ntpath.py

```diff
@@ -299,24 +299,24 @@ def isjunction(path):
     from nt import _getvolumepathname
 except ImportError:
     _getvolumepathname = None
+
```

Comment:
Please revert this change. Any stylistic or formatting changes would have to go in their own PR, if we accepted them at all.